src/WebJobs.Script/Description/DotNet/ScriptRuntimeAssembly.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal sealed class ScriptRuntimeAssembly
+    {
+        public string Name { get; set; }
+
+        public string ResolutionPolicy { get; set; }
```

Comment:
Can you make this an enum so it'd fail when loading the json file? It looks like if there was a misspelling it may fail later during runtime otherwise...

Reply:
Keeping the load policy names as is for now. This will throw with a very specific error message.

These changes lift some of the restrictions we had in place with the initial implementation (particularly on System.* assemblies) and introduce better unification support.